Ninano/Ninano/Views/EventTab/SerchResult/PerformancesViewCell.swift

```diff
@@ -9,28 +9,58 @@ import UIKit
 
 final class PerformancesViewCell: UICollectionViewCell {
     
-    @IBOutlet var performanceImage: UIImageView!
+    @IBOutlet var eventImage: UIImageView!
```

Comment:
지금 명명이면 UIImage로 생각될 것 같아요 .. ! 조금 더 명확해지면 좋을 것 같아요 ~
```suggestion
    @IBOutlet var eventImageView: UIImageView!
```